src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -623,6 +634,19 @@ else if( Character.isDigit(text.charAt(i)) )
     }
   }
 
+  /**
+   * Returns a BufferedImage containing the contents of `text` rendered on one line, on a transparent background, with no scaling applied.
+   */
+  public static BufferedImage getTextAsImage(String text)
+  {
+    Sprite letters = getLettersSmallCaps();
+    int width = letters.getFrame(0).getWidth() * text.length();
+    int height = letters.getFrame(0).getHeight();
+    BufferedImage textImage = createTransparentSprite(width, height);
+    drawTextSmallCaps(textImage.getGraphics(), text, 0, 0, 1);
```

Comment:
Why always `SmallCaps`? I feel like stuff is more readable with upper and lower-case letters.
Also in the case of CO names, the capitalization is important for knowing what to name their images, heh.

Reply:
I think I mostly just did that since you used small caps for the current CO selection menu. No reason we can't allow both.